src/test/java/com/epam/training/array/ArraySorterTest.java

```diff
@@ -1,13 +1,38 @@
 package com.epam.training.array;
 
-public class ArraySorterTest {
+import com.epam.training.array.impl.BubbleArraySorter;
+import com.epam.training.array.impl.NegativeNumbersArraySorter;
+import com.epam.training.array.impl.PositiveNumbersArraySorter;
+import org.junit.Assert;
+import org.junit.Test;
 
-    //TODO: uncomment test when junit dependency is added
-    //@Test
-    public void testSortShouldSortWhenPositiveNumbers(){
+public class ArraySorterTest {
+    private ArraySorter arraySorter;
 
+    @Test
+    public void testSortShouldSortWhenPositiveNumbers() {
+        Array array = new Array(new int[]{-3, -5, 34, 67, 22, 5, -20, 84, 44});
+        Array expectedArray = new Array(new int[]{5, 22, 34, 44, 67, 84});
+        arraySorter = new PositiveNumbersArraySorter();
+        Array actualArray = arraySorter.sort(array);
+        Assert.assertEquals(expectedArray, actualArray);
     }
 
-    //TODO: add tests for other positive and negative scenarios
+    @Test
+    public void testSortShouldSortWhenNegativeNumbers() {
+        Array array = new Array(new int[]{-4, -8, 10, -1, 0, 20, -50, -39, 30, -7, -20});
+        Array expectedArray = new Array(new int[]{-50, -39, -20, -8, -7, -4, -1});
+        arraySorter = new NegativeNumbersArraySorter();
+        Array actualArray = arraySorter.sort(array);
+        Assert.assertEquals(expectedArray, actualArray);
+    }
 
-}
+    @Test
+    public void testBubbleSort() {
+        Array array = new Array(new int[]{3, 6, 7, 8, 9, 2, 3, 5, 79});
+        Array expectedArray = new Array(new int[]{2, 3, 3, 5, 6, 7, 8, 9, 79});
+        arraySorter = new BubbleArraySorter();
+        Array actualArray = arraySorter.sort(array);
+        Assert.assertEquals(expectedArray, actualArray);
+    }
+}
```

Comment:
@AIniuseva Could you add new line please

Reply:
@NikitaUstsiushenka Done. Please, see the change here: https://github.com/AIniuseva/array-task/pull/1/files#diff-3add5414a65507e37336487ecff2896a84071c4cf573330cfa5bab939acdeab2R46